CHANGELOG.rst

```diff
@@ -1,6 +1,9 @@
 Unreleased
 ----------
 
+* Add tests for Wagtail 6.1 (@katdom13)
+* Drop support for Django 4.2 (@katdom13)
```

Comment:
**Question:** Do you mean 3.2?

Reply:
@tm-kn ,
Thanks! I actually meant Django < 4.2 but forgot the sign.
I also fixed the erring tests.

Reply:
Thanks! 🙌